Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml

```diff
@@ -0,0 +1,76 @@
+# Uses the chessboard and generic pieces
+- type: entity
+  parent: BaseItem
+  id: CheckerBoard
+  name: checkerboard
+  description: A checkerboard. Pieces included!
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Tabletop/chessboard.rsi
+    state: chessboard
+  - type: TabletopGame
+    boardName: tabletop-checker-board-name
```

Comment:
this isnt properly localized

Reply:
What does that mean?

Reply:
Mirrorcult is saying, you didn't properly localize what tabletop-checker-board-name is. You should add the correct localization and get it from tabletop.ftl. Look at home the other board games do it.